ProcessMaker/screen-builder
# FOUR-17472 Fix reference to _parent in MultiInstance Tasks

## Issue & Reproduction Steps

Can not access to _parent in tasks in MultiInstance Tasks

Expected behavior: 

MultiInstance Task must be able to acces to _parent

Actual behavior: 

MultiInstance Task must be able to acces to _parent

## Solution
- Fix reference to _parent in MultiInstance Tasks

## How to Test
- Import the attached process
- Run the process
- Follow the process, upload some files (images recommended) to the process
- Continue to the MultiInstance Process
- Verify the files you uploaded are visible

https://github.com/user-attachments/assets/cf291e2f-95a0-4c81-bb9c-1a607df86d0d

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-17472

## Code Review Checklist

ci:next
ci:vue-form-elements:next
